lib/zendesk_apps_tools/server.rb

```diff
@@ -4,7 +4,6 @@
 module ZendeskAppsTools
   class Server < Sinatra::Base
     set :protection, :except => :frame_options
-    set :public_folder, proc { "#{settings.root}/assets" }
```

Comment:
You could do something like this tho, https://github.com/britg/sinatra-cross_origin